department-of-veterans-affairs/vets-website
# [SCREENREADER]: I have read the opt out policy checkbox needs "Required" added to label

## Description
[SCREENREADER]: I have read the opt out policy checkbox needs "Required" added to label

Steps to test:

1. Go to http://localhost:3001/education/opt-out-information-sharing/opt-out-form-0993/review-and-submit
2. The form field should have the (*Required) on it.

## Testing done
* Local Testing

## Related Issue
* https://github.com/department-of-veterans-affairs/vets.gov-team/issues/13828

## Screenshots
<img width="762" alt="screen shot 2019-01-17 at 11 34 25 pm" src="https://user-images.githubusercontent.com/5377648/51372180-327b8480-1ab1-11e9-9822-7acfed7ecb56.png">

## Acceptance criteria
- [ ] Should have a `(*Required)` in the form

## Definition of done
